prism-highlighter.html

```diff
@@ -59,7 +59,12 @@
         return;
       }
 
+      if(event.detail._highlighted === true) {
```

Comment:
In actual fact, @ashi009 stopPropagation isn't sufficient here. See this [example](http://codepen.io/anon/pen/rLavWx?editors=1010).

If you have two listeners (in this case two `prism-highlighter` under the same parent), they will both be hit as `stopPropagation` only prevents further bubbling upwards.